aspnetcore/getting-started/index.md

```diff
@@ -70,8 +70,6 @@ This command might prompt you for your password to install the certificate on th
 
 # [Linux](#tab/linux)
 
-For Windows Subsystem for Linux, see [Trust HTTPS certificate from Windows Subsystem for Linux](xref:security/enforcing-ssl#wsl).
-
 See the documentation for your Linux distribution on how to trust the HTTPS development certificate.
```

Comment:
One ❓ tho ........ does the command that you gave work on the **supported** Linux distros? If so, this could say ....

(I can't do code in a comment with triple-backslashes in a suggestion ... the following is a cut-and-paste from this comment) ...

> For supported Linux distributions, install the development certificate by executing the following command in a command shell:
> 
> \`\`\`console
> dotnet tool install --global dotnet-dev-certs
> \`\`\`